provider/thp_fragment_tool.py

```diff
@@ -0,0 +1,45 @@
+"""
+thp_fragment tool.
+
+This module is meant to copy, build and execute the thp_fragment tool.
+"""
+import os
+import shutil
+
+from avocado.utils import process
+
+from virttest import data_dir
+
+
+dst_dir = "/var/tmp"
+test_bin = "/var/tmp/thp_fragment"
+source_file = "thp_fragment.c"
+
+
+def clean():
+    process.system("rm -rf %s %s" % (test_bin, source_file))
```

Comment:
`process.system` does not support `cwd` feature, so before deleting them, we should ensure in the tmp dir, or just use test.tmpdir instead.

Reply:
Updated !

Reply:
```
[stdlog] 2024-04-09 05:33:25,190 avocado.utils.process process          L0657 INFO | Running 'rm -rf /var/tmp/thp_fragment /var/tmp/thp_fragment.c'
[stdlog] 2024-04-09 05:33:25,195 avocado.utils.process process          L0714 INFO | Command 'rm -rf /var/tmp/thp_fragment /var/tmp/thp_fragment.c' finished with 0 after 0.000845756s
```